Command/Validators.php

```diff
@@ -127,6 +127,19 @@ public static function validateEntityName($entity)
         return $entity;
     }
 
+    public static function validateEntityPropertyName($property)
+    {
+        if (in_array($property, self::getReservedWords())) {
```

Comment:
if we use this, we shall remove the isReservedKeyword()-method, but this is a BC